### ISSUE TITLE
Copy Paste results in double pasting of copied text

### ISSUE THREAD

**Issue by dvd9604** [notable/notable#968]
- **OS Version**: Ubuntu 18.04
- **Notable Version**: 1.8.2

### Current behavior
Pasting results in the copied text being pasted twice back to back. 

1. Split Pane View
2. highlight then CTRL + C
3. CTRL + V

### Expected behavior

Paste copied item only once.

**Comment by fabiospampinato** [comment]
Duplicate of #779, it's been fixed and an update with this fix and others will be pushed soon, I've been more busy than I anticipated in the past couple of weeks so there's been some delay.
